doc/getting_started.md

```diff
@@ -0,0 +1,109 @@
+# Getting started
+
+<!-- TOC -->
+
+- [Getting started](#getting-started)
+    - [SPIRE artifacts](#spire-artifacts)
+    - [Building the artifacts](#building-the-artifacts)
+    - [Installing your artifact in Ubuntu 16.04](#installing-your-artifact-in-ubuntu-1604)
+    - [Configuration](#configuration)
+    - [Running SPIRE with join token](#running-spire-with-join-token)
+
+<!-- /TOC -->
+
+## SPIRE artifacts
+
+You can either download the binaries from [here](https://github.com/spiffe/spire/releases) or build them by
+[setting up your dev environment first](https://github.com/spiffe/spire/blob/master/CONTRIBUTING.md).
+
+## Building the artifacts
```

Comment:
As per above, suggest removing this section, but instead describing in detail how to use the pre-built binaries.